extensions/rich_text_components/Math/directives/oppia-noninteractive-math.directive.ts

```diff
@@ -74,12 +74,15 @@ angular.module('oppia').directive('oppiaNoninteractiveMath', [
             // don't pre-load the images in this case. So we directly assign
             // the url to the imageUrl.
             try {
+              ctrl.imageUrl = null;
               if (
                 ContextService.getImageSaveDestination() ===
-                IMAGE_SAVE_DESTINATION_LOCAL_STORAGE) {
+                IMAGE_SAVE_DESTINATION_LOCAL_STORAGE && (
+                  ImageLocalStorageService.isInStorage(svgFilename))) {
                 ctrl.imageUrl = ImageLocalStorageService.getObjectUrlForImage(
                   svgFilename);
-              } else {
+              }
+              if (ctrl.imageUrl === null) {
```

Comment:
Why we use `null` here and `''` in the file above?

Reply:
Made it consistent.